rest_framework/serializers.py

```diff
@@ -1081,7 +1081,10 @@ def save_object(self, obj, **kwargs):
                     self.save_object(related)
                 else:
                     # Reverse FK or reverse one-one
-                    setattr(obj, accessor_name, related)
+                    try:
+                        setattr(obj, accessor_name, related)
+                    except ValueError:
+                        getattr(obj, accessor_name).add(*related, bulk=False)
```

Comment:
If we don't try to update our code now, when? It is so easy to put this off. I'm afraid we can be going down the dark side...